k8s-bundle.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: dd-cf-agent
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: dd-cf-agent
+    spec:
+      containers:
+      - name: dd-cf-agent
+        image: registry.honestbee.com/honestbee/cloudflare-datadog:latest
```

Comment:
I think we should drop the k8s-bundle.yaml and refer to the helm chart? (Unless we want to support ppl who don't want to install helm?)